cinema/migrations/0002_movie_image.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 4.1 on 2024-10-31 06:38
+
+import cinema.models

```

Comment:
The import statement `import cinema.models` is not used directly in the code. Consider importing only the `movie_image_path` function directly if that's the only part being used, like `from cinema.models import movie_image_path`.